_examples/range/range.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
I actually missed the middleware part in the `mux` example. Forget what I said :wink: